card/mod/bootstrap/set/abstract/bootswatch_theme/thumbnail.haml

```diff
@@ -1,10 +0,0 @@
-.card.mb-4.box-shadow
```

Comment:
there is no thumbnail_image view in decko now.  I think this haml file must have been obviated?